FlowCal/gate.py

```diff
@@ -275,13 +275,13 @@ def density2d(data,
     gate_fraction : float, optional
         Fraction of events to retain after gating.
     xscale : str, optional
-        Scale of the bins generated for the x axis. Can be either
-        ``linear`` or ``log``. `xscale` is ignored in `bins is an array or
-        a list of arrays.
+        Scale of the bins generated for the x axis, either ``linear``,
+        ``log``, or ``logicle``. `xscale` is ignored in `bins is an array
+        or a list of arrays.
     yscale : str, optional
-        Scale of the bins generated for the y axis. Can be either
-        ``linear`` or ``log``. `yscale` is ignored in `bins is an array or
-        a list of arrays.
+        Scale of the bins generated for the y axis, either ``linear``,
+        ``log``, or ``logicle``. `yscale` is ignored in `bins is an array
+        or a list of arrays.
     sigma : scalar or sequence of scalars, optional
```

Comment:
Oh, just for the same reasons that we went with logicle for the Excel UI. Seems more appropriate in the general case. And in an effort to try and standardize what kind of plot you're looking at. Makes sense to make one kind of plot the default everywhere. My feelings aren't super strong on this, though.